### PR TITLE
Create color-to-hex-argb-string-converter.md

### PR DESCRIPTION
**Pull request by cl2raul66**
Documentacion de ColorToHexRgbaStringConverter

### CODE REVIEW

**Comment by cl2raul66** [comment]
@microsoft-github-policy-service agree

File: docs/maui/converters/color-to-hex-argb-string-converter.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+                        converter: new ColorToHexArgbStringConverter())
+            }
+        };
+    }
+}
+```
+
+## Examples 
+
+You can find an example of this converter in action in the .NET MAUI Community Toolkit Sample Application.
```
Comment (bijington) [inline docs/maui/converters/color-to-hex-argb-string-converter.md:112]:
```suggestion
[.NET MAUI Community Toolkit Sample Application](https://github.com/CommunityToolkit/Maui/blob/main/samples/CommunityToolkit.Maui.Sample/Pages/Converters/ColorsConverterPage.xaml)
```

File: docs/maui/converters/color-to-hex-argb-string-converter.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+## API
+
+You can find the source code for `ColorToHexArgbStringConverter` over on the .NET MAUI Community Toolkit GitHub repository.
```
Comment (bijington) [inline docs/maui/converters/color-to-hex-argb-string-converter.md:116]:
```suggestion
You can find the source code for `ColorToHexArgbStringConverter` over on the [.NET MAUI Community Toolkit GitHub repository](https://github.com/CommunityToolkit/Maui/blob/main/src/CommunityToolkit.Maui/Converters/ColorToStringConverter.shared.cs).
```

**Review by bijington — APPROVED** [review]
@cl2raul66 thank you for this. I have just added 2 suggestions for the correct links which I can commit after this and we should be good.

**Comment by bijington** [comment]
I've just marked this as Draft so it doesn't get accidentally merged until we release the new converter

**Pull request merged**